runtime/shiden/src/lib.rs

```diff
@@ -748,32 +756,39 @@ pub type Executive = frame_executive::Executive<
     frame_system::ChainContext<Runtime>,
     Runtime,
     AllPalletsWithSystem,
-    DappsStakingFixEraLength,
+    (InitRewardConfigSettings,),
 >;
 
 // Migration for fixing era length in dapps staking.
-pub struct DappsStakingFixEraLength;
+pub struct InitRewardConfigSettings;
 
-impl OnRuntimeUpgrade for DappsStakingFixEraLength {
+impl OnRuntimeUpgrade for InitRewardConfigSettings {
     fn on_runtime_upgrade() -> frame_support::weights::Weight {
-        use frame_system::pallet::Config;
-        use pallet_dapps_staking::{Config as DappStakingConfig, CurrentEra, NextEraStartingBlock};
-        use sp_runtime::SaturatedConversion;
-
-        let dapps_staking_block_offset: u32 = 489600; // calculated 2022/03/29
-                                                      // Shiden: 1384490 - (1384490 % 7200) - (124 * 7200)
-
-        let blocks_per_era =
-            <Runtime as DappStakingConfig>::BlockPerEra::get().saturated_into::<u32>();
-        let current_era = CurrentEra::<Runtime>::get();
+        // TODO: discuss these params on sync & with community
```

Comment:
is this still TODO?

Reply:
Yes, we discussed it on retro yesterday.

Doesn't have to be done prior to merge since we're going with legacy distribution config but I wanted to keep a reminder here.